.jenkins/pytorch/test.sh

```diff
@@ -401,9 +412,21 @@ else
   test_benchmarks
   test_rpc
   if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
+    # code coverage for python sources
     pushd test
-    echo "Generating XML coverage report"
+    echo "Generating XML coverage report for python sources"
     time python -mcoverage xml
     popd
+
+    # code coverage for c++ sources
+    pushd build
+    echo "Generating lcov coverage report for c++ sources"
+    time lcov --capture --directory . --output-file coverage.info
+    # remove files that we do not care about from coverage report
+    time lcov --remove coverage.info '/usr/*' '*/third_party/*' '*/caffe2/*' --output-file coverage.info
+    # remove .AVX2.cpp suffix and remove build/ in filename in coverage.info
+    time python ../tools/code_coverage/package/tool/modify_coverage_info.py coverage.info
```

Comment:
This better be done using `fixes:` section of https://github.com/pytorch/pytorch/blob/master/codecov.yml

Reply:
I didn't find a way to remove `*.AVX2.cpp`. Besides, this script may also be used locally for `gcc` coverage report in the future.